client/controllers/public/home.js

```diff
@@ -1,9 +1,59 @@
 Template.home.onCreated(function() {
-  Meteor.subscribe('latestChannels');
+  var self = this;
+  self.autorun(function() {
+    Meteor.subscribe('latestChannels');
+    if (Meteor.userId()) {
+      Meteor.subscribe('myChannels');
+    }
```

Comment:
I wanted to try to use a template instance variable here for the user, but there is a consensus that Meteor is anti-dry in this respect.

https://forums.meteor.com/t/access-parent-template-reactivevar-from-child-template/973/15